.storybook/main.js

```diff
@@ -16,30 +15,43 @@ module.exports = {
     "@storybook/addon-a11y",
     "@chakra-ui/storybook-addon",
   ],
-  framework: "@storybook/react",
+  framework: {
+    name: "@storybook/react-webpack5",
+    options: {},
+  },
   refs: {
     "@chakra-ui/react": {
       disable: true,
     },
   },
-  core: {
-    builder: "webpack5",
-  },
   features: {
     storyStoreV7: true,
     emotionAlias: false,
```

Comment:
Are these two options needed on v7?

Reply:
@pettinarip

Both can be removed. Per migration docs `storyStoreV7` is set to true by default, and `emotionAlias` is no longer needed.

Reply:
Removed.